gatling-http/src/main/scala/io/gatling/http/ahc/HttpEngine.scala

```diff
@@ -73,27 +73,33 @@ class HttpEngine(
 
       import httpComponents._
 
+      val debugEnabled = logger.underlying.isDebugEnabled
```

Comment:
this code will be only executed once, hence no need to have a val for this, just directly test `logger.underlying.isDebugEnabled` below

Reply:
done, not sure why I introduced it in the first place